### PR TITLE
Playlist Regions

### PR DESCRIPTION
**Pull request by Legoman99573**
Its done and tested. Should work 😅

### CODE REVIEW

File: Plugin source code/pom.xml [hunk 1]
```diff
@@ -99,13 +99,39 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.bukkit</groupId>
+            <artifactId>bukkit</artifactId>
+            <version>1.12.2-R0.1-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.bukkit</groupId>
+            <artifactId>bukkit</artifactId>
+            <version>1.12.2-R0.1-SNAPSHOT</version>
```
Comment (ApocalypsjeNL) [inline Plugin source code/pom.xml]: Why repeating bukkit import?

File: Plugin source code/pom.xml [hunk 2]
```diff
@@ -99,13 +99,39 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
             <version>1.16.16</version>
-            <scope>provided</scope>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/lombok.jar</systemPath>
```
Comment (ApocalypsjeNL) [inline Plugin source code/pom.xml]: Why added Lombok jar

File: Plugin source code/pom.xml (hunk 1 above)
Comment (Legoman99573) [reply Plugin source code/pom.xml]: gonna fix that up
idk why it duplicated tbh

**Comment by Legoman99573** [comment]
@ApocalypsjeNL Check commits now. Loaded from repo instead and fixed org.bukkit duplication. Still dont know how it did that importing everything.

**Pull request merged**